package.json

```diff
@@ -27,5 +27,9 @@
     "github": "^7.0.0",
     "gulp": "^3.9.1",
     "yargs": "^6.5.0"
+  },
```

Comment:
Hi!  Just want to clarify
I should remove:
"github": "^7.0.0",
"gulp": "^3.9.1",
"yargs": "^6.5.0"